sql/scan.go

```diff
@@ -725,6 +740,32 @@ func (v *qnameVisitor) getDesc(qname *parser.QualifiedName) *TableDescriptor {
 	return nil
 }
 
+// maybeExpandCount provides the special handling for COUNT(*) and
+// COUNT(foo.*), expanding the star into a tuple containing the columns in the
+// referenced table.
+func (v *qnameVisitor) maybeExpandCount(qname *parser.QualifiedName) parser.Expr {
```

Comment:
by the way, why does `COUNT(*)` need to expand to `COUNT(columns...)`? Can it expand to `COUNT(primaryKey)` instead?